unitTests/ui/FoldingTest.ts

```diff
@@ -217,6 +217,19 @@ export function FoldingTest() {
         queryData = null;
       });
 
+      it('should trap the error if a query error happens when getting more results', () => {
```

Comment:
Would it be better to check that the method returns an empty array rather than whether or not it throws? My thinking is if someone were to remove the `return` statement, this test would still pass.

Reply:
Good point, but I'll add a new test for the empty array check, since checking that it does not "pollute" the console needlessly is still a valid test.